config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
```

Comment:
I don't understand why you would need this if you already have your own runtime binary, `runv` `lkvm` that is starting the thing.  Why is this needed?